cmd/cli/extra.go

```diff
@@ -30,3 +30,27 @@ func enablePlottingInAllSubscriptions(logger moira.Logger, database moira.Databa
 	}
 	return nil
 }
+
+func makeAllTriggersDataSourceLocal(logger moira.Logger, database moira.Database) error {
```

Comment:
May be not needed in master ? Function will be dangerous. when running on production.

Reply:
done https://github.com/moira-alert/moira/pull/725